boot/boot.js

```diff
@@ -1739,17 +1744,23 @@ $tw.modules.define("$:/boot/tiddlerdeserializer/dom","tiddlerdeserializer",{
 
 $tw.loadTiddlersBrowser = function() {
 	// In the browser, we load tiddlers from certain elements
-	var containerIds = [
-		"libraryModules",
-		"modules",
-		"bootKernelPrefix",
-		"bootKernel",
-		"styleArea",
-		"storeArea",
-		"systemArea"
+	var containerSelectors = [
+		// IDs for old-style v5.1.x tiddler stores
+		"#libraryModules",
+		"#modules",
+		"#bootKernelPrefix",
+		"#bootKernel",
+		"#styleArea",
+		"#storeArea",
+		"#systemArea",
+		// Classes for new-style v5.2.x JSON tiddler stores
+		".tiddlywiki-tiddler-store"
```

Comment:
Just a thought: Would it be better to be more specific here, e.g. `"html>body>script.tiddlywiki-tiddler-store"`?

I'm imagining some fun behaviour if someone manages to add that class somewhere it's not expected.

(It's out of scope for this PR, but the same question could be asked about the the other selectors. E.g. maybe `html>body>div#storeArea` is better than `#storeArea`.)

Reply:
As I did understand that, it should be possible to simply add more SCRIPT tags to an existing html file by appending it at the end of the file. .. So those scripts will be outside the main HTML tabs. 

eg: 

```
cat newStuff >> ./path/tiddlywiki.html
```

Where `newStuff` contains: 

```
<script class="tiddlywiki-tiddler-store" type="application/json">[
{"title":"A","text":"One"},
{"title":"B","text":"Two"}
]</script>
```

@Jermolene Did I get this wrong?

Reply:
Hmm, that's an interesting use case. There might be some disadvantages related to the doc being malformed, but I don't have a strong opinion on it. FWIW Tiddlyhost inserts its extra script element after the existing one.

Reply:
If the user arrange a raw markup section containing a tag with the magic classes/IDs, then things could go wrong, but otherwise things seem safe.